execution/evm/abi/abi.go

```diff
@@ -1165,7 +1166,7 @@ func (abiSpec *AbiSpec) UnpackWithID(data []byte, args ...interface{}) error {
 	})
 }
 
-func (abiSpec *AbiSpec) Pack(fname string, args ...interface{}) ([]byte, error) {
+func (abiSpec *AbiSpec) Pack(fname string, args ...interface{}) ([]byte, bool, error) {
```

Comment:
Can we have a doc comment on this, particularly for what the return values are

Reply:
Add godoc